samples/apps/logging/logging_schema.h

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 #include "ds/json.h"
+#include "node/rpc/call_types.h" // For GetReceipt
```

Comment:
That seems less narrowly scoped than it could, why not ccf/receipt.h?

Reply:
Absolutely - that's now done [here](https://github.com/microsoft/CCF/pull/2328/files#diff-35569e31aa8a27272198e8492dc20ef17a2731da8ff3bb337c44e500ca28471eR4).